core/object/object.cpp

```diff
@@ -1313,8 +1313,14 @@ void Object::get_signals_connected_to_this(List<Connection> *p_connections) cons
 Error Object::connect(const StringName &p_signal, const Callable &p_callable, uint32_t p_flags) {
 	ERR_FAIL_COND_V_MSG(p_callable.is_null(), ERR_INVALID_PARAMETER, "Cannot connect to '" + p_signal + "': the provided callable is null.");
 
-	Object *target_object = p_callable.get_object();
-	ERR_FAIL_NULL_V_MSG(target_object, ERR_INVALID_PARAMETER, "Cannot connect to '" + p_signal + "' to callable '" + p_callable + "': the callable object is null.");
+	if (p_callable.is_standard()) {
+		// NOTE: This branch should probably removed in favor of the `is_valid` branch, but there exist some classes
```

Comment:
```suggestion
		// FIXME: This branch should probably removed in favor of the `is_valid` branch, but there exist some classes
```
Or TODO.
NOTE doesn't signify that it's something actionable.

Reply:
Ah, I like the idea of using something that marks this as actionable, will change!